manifest.json

```diff
@@ -2,7 +2,7 @@
     "manifest_version": 2,
     "name": "Moonlight Game Streaming",
     "short_name": "Moonlight",
-    "version": "0.10.21",
+    "version": "0.10.22",
```

Comment:
For future PRs, please don't increment the version. I'll commit that when I do a release.